src/SecurityDetails/Converters/MetaConverter.ts

```diff
@@ -0,0 +1,152 @@
+/* *
+ *
+ *  (c) 2009-2025 Highsoft AS
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ *  Authors:
+ *  - Mateusz Bernacik
+ *
+ * */
+
+
+'use strict';
+
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+
+import {
+    SecurityDetailsConverterOptions,
+    SecurityDetailsMetadata
+} from '../SecurityDetailsOptions';
+import SecurityDetailsJSON from '../SecurityDetailsJSON';
+import SecurityDetailsConverter from '../SecurityDetailsConverter';
+
+/* *
+ *
+ *  Class
+ *
+ * */
+
+
+export class MetaConverter extends SecurityDetailsConverter {
+
+
+    /* *
+     *
+     *  Constructor
+     *
+     * */
+
+
+    public constructor (
+        options?: SecurityDetailsConverterOptions
+    ) {
+        super(options);
+
+        this.metadata = {
+            columns: {}
+        };
+    }
+
+
+    /* *
+     *
+     *  Properties
+     *
+     * */
+
+
+    public readonly metadata: SecurityDetailsMetadata;
+
+
+    /* *
+     *
+     *  Functions
+     *
+     * */
+
+
+    public override parse (
+        options: SecurityDetailsConverterOptions
+    ): void {
+        const metadata = this.metadata;
+        const table = this.table;
+        const userOptions = {
+            ...this.options,
+            ...options
+        };
+        const json = userOptions.json;
+
+        // Validate JSON
+
+        if (!SecurityDetailsJSON.isSecurityDetailsResponse(json)) {
+            throw new Error('Invalid data');
+        }
+
+        // Prepare table
+
+        table.deleteColumns();
+        table.setColumn('Meta');
+        table.setColumn('Value');
+
+        if (json.length) {
+            const securityDetails = json[0];
+
+            const rows = [
```

Comment:
Q: Does it make sense to add CategoryBroadAssetClass ID & Name? And FundAttributes?

![image](https://github.com/user-attachments/assets/7b735c5a-e256-4481-9206-785133135570)

Reply:
And `ActualManagementFree`?